### PR TITLE
Update Redux example to match the 1.0.0-rc API

### PR DESCRIPTION
**Pull request by rwillrich**
I've updated the Redux example to reflect the API and naming changes on 1.0.0-rc.

### CODE REVIEW

**Comment by gaearon** [comment]
To clarify, we wanted to move to `stage: 2` so people don't freak out about ES7 features that aren't baked yet.

**Comment by rwillrich** [comment]
@gaearon Yeah, forgot to explain this.

**Comment by voronianski** [comment]
@rwillrich @gaearon thanks, merging

File: redux/js/components/ProductsContainer.jsx [hunk 1]
```diff
@@ -18,26 +17,23 @@ class ProductItemContainer {
...
         );
     }
 }
 
-@connect(state => ({
-    // named in stores/index.js
-    products: state.products
-}))
-export default class ProductsListContainer {
+const ConnectedProductItemContainer = connect(() => ({}))(ProductItemContainer);
```
Comment (voronianski) [inline redux/js/components/ProductsContainer.jsx:27]: @rwillrich IMHO this looks a bit weird :)

Comment (rwillrich) [inline redux/js/components/ProductsContainer.jsx:27]: We changed to Babel `stage: 2` so we have to manually apply decorators. But I'll check if I can do it better :P

**Pull request merged**